src/net.cpp

```diff
@@ -1262,28 +1264,10 @@ void CConnman::InactivityCheck(CNode *pnode)
     }
 }
 
-void CConnman::SocketHandler()
+bool CConnman::GenerateSelectSet(std::set<SOCKET> &recv_set, std::set<SOCKET> &send_set, std::set<SOCKET> &error_set)
```

Comment:
I guess we *want* these sets to be sorted by fd, or at least predictably sorted, as it's iterated over--so using `set` over `unordered_set` makes sense

Reply:
regardless of iterating i cant imagine we'd see any change in a benchmark here

Reply:
It's probably slightly more efficient to return sorted vectors, and use `std::binary_search` to look up things in it (as `std::map` needs one allocation per element), but agree that it is unlikely to matter here.